editor/templates/place/edit.html

```diff
@@ -212,6 +262,64 @@ <h3>Editing {{ wof_doc.properties['wof:name'] or wof_doc.id }}</h3><small>(<a hr
 </div>
 
 <form class="needs-validation" novalidate action="{{ url_for('place.edit_place', url=wof_url) }}" method="POST">
+    <h4>Geometries</h4>
+    <!-- https://github.com/whosonfirst/whosonfirst-properties/blob/master/properties/wof/name.json -->
+    <div class="form-group">
+        {%- if wof_doc.properties['geom:area_square_m'] is defined %}
+        <small id="wof-name-help" class="form-text text-muted">Land-only multi-polygons are preferred, points are allowed. {{ wof_doc.properties['geom:area_square_m'] }} square meters</small>
+        {%- else %}
+        <small id="wof-name-help" class="form-text text-muted">Land-only multi-polygons are preferred, points are allowed.</small>
+        {%- endif %}
+        <input type="checkbox" id="geom_default" name="geom_default" value="Default geom" disabled=True checked=True>
+        <label for="geom_default">Default geometry</label>&nbsp;<small>(<code>geometry</code>)</small>
+        {%- if wof_doc.properties['src:geom_alt'] is defined %}
+        <small id="wof-name-help" class="form-text text-muted">Alternate geometries are present from the following {{ wof_doc.properties['src:geom_alt']|length }} sources.</small>
+        <input type="checkbox" id="geom_alternates" name="geom_alternates" value="Alternate geometries" disabled=True checked=True>
+        <label for="geom_default">Alternate geometries</label>&nbsp;<small>(<code>src:geom_alt</code>)</small>
+        {%- for alt_geom in wof_doc.properties['src:geom_alt'] %}
+        <ul class="form-text text-muted">
+            <li class="col-md-2 col-form-label">{{ alt_geom }}</li>
```

Comment:
The CSS styling here is too big in the list. Ideally it's "small", or instead of the list it's a series of inline text chunks at the end of the label line above (with special handling for commas & etc).